velox/functions/lib/DateTimeFormatter.cpp

```diff
@@ -347,6 +347,126 @@ int64_t parseTimezone(const char* cur, const char* end, Date& date) {
   return -1;
 }
 
+// Contains a list of all time zone names in a convenient format for searching.
```

Comment:
see my comment below, but should we encapsulate this logic within the TimeZone object? So these things could be exposed under a more convenient API like

```
timezone->name();
timezone->shortName();
```